libvmi/os/windows/windows.h

```diff
@@ -23,6 +23,10 @@
 
 #include "private.h"
 
+#define LOWEST_KERNEL_ADDRESS_64BIT 0xffff800000000000
+#define LOWEST_KERNEL_ADDRESS_32BIT 0x80000000
```

Comment:
So IMHO what would make sense is to reduce this PR to just the parts that introduce the ntoskrnl_va config option and adjusting the init process to skip the parts where the kernel is being searched for.